openwisp_controller/config/models.py

```diff
@@ -97,11 +101,11 @@ class Meta(AbstractDevice.Meta):
         abstract = False
 
 
-class Config(OrgMixin, TemplatesVpnMixin, AbstractConfig):
+class Config(TemplatesVpnMixin, AbstractConfig):
     """
     Concrete Config model
     """
-    device = models.OneToOneField('config.Device', on_delete=models.CASCADE)
+    device = models.OneToOneField('config.Device', on_delete=models.CASCADE, null=False, blank=False)
```

Comment:
`null=False, blank=False` are implicit, I prefer if we omit this change because is not needed